sonar-xml-plugin/src/main/java/org/sonar/plugins/xml/checks/XmlFile.java

```diff
@@ -70,6 +70,10 @@ private void checkForCharactersBeforeProlog(FileSystem fileSystem) {
       Pattern firstTagPattern = Pattern.compile("<[a-zA-Z?]+");
 
       for (String line : Files.readLines(inputFile.file(), fileSystem.encoding())) {
+        if (lineNb == 1 && line.length() > 0 && line.charAt(0) == '\ufeff') {
```

Comment:
@vilchik-elena minor: this could be slightly simplified to `lineNb == 1 && line.startsWith("\ufeff")`